pkg/controller/node/rate_limited_queue.go

```diff
@@ -221,7 +219,7 @@ func (q *RateLimitedTimedQueue) Try(fn ActionFunc) {
 			val.ProcessAt = now.Add(wait + 1)
 			q.queue.Replace(val)
 		} else {
-			q.queue.RemoveFromQueue(val.Value)
+			q.queue.Remove(val.Value)
```

Comment:
Maybe I can use pod.UID instead of  podNamespace/podName" when queueing a pod for eviction, discard the change here, and add some logic to call Remove() when received pod delete event?